lib/iiif_print.rb

```diff
@@ -155,6 +155,10 @@ def self.default_fields(fields: config.metadata_fields)
   end
 
   def self.fields_for_allinson_flex(fields: allinson_flex_fields)
+    sort_order = IiifPrint.config.iiif_metadata_sorter_fields
+    fields = sort_af_fields!(fields, sort_order) if sort_order
```

Comment:
My read is that we're doing about four loops through the fields.  This works, but am wondering about performance.